packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:rx_bloc/rx_bloc.dart';
 import 'package:rxdart/rxdart.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
```

Comment:
Do you need the entire package ? If not please import only the library you actually need.

Reply:
When I try to use only the biometrics_setting_message_type.dart like this:
 import 'package:widget_toolkit_biometrics/src/models/biometrics_setting_message_type.dart'; there is an analysis error: https://dart-lang.github.io/linter/lints/implementation_imports.html
 What I found that  I can do to avoid access to the other content of the widget_toolkit_biometrics package in this file is this:
import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart' show BiometricsMessage;

Or export the file needed as a library from the biometrics package:
biometrics_setting_message_type.dart:
library biometrics_setting_message_type;
export 'package:widget_toolkit_biometrics/src/models/biometrics_setting_message_type.dart';

Reply:
My bad. There is just one library https://pub.dev/documentation/widget_toolkit_biometrics/latest/widget_toolkit_biometrics/widget_toolkit_biometrics-library.html so I'll resolve this comment